package.json

```diff
@@ -48,7 +48,6 @@
     "rimraf": "^5.0.1",
     "ts-jest": "^29.1.1",
     "ts-node": "^10.9.1",
-    "tslib": "^2.6.1",
```

Comment:
The `tslib` dependency was only needed because of `tsup`. Now that it's no longer a dependency we can remove it.